README.md

```diff
@@ -15,6 +15,17 @@ go get github.com/nats-io/nats.go/
 go get github.com/nats-io/nats-server
 ```
 
+When using Go modules (`GO111MODULE=on`):
```

Comment:
Go 1.12 makes them on by default I think, so maybe say "When using or transitioning to Go Modules support, ..."